### PR TITLE
Fixes github issue 1777

### PR DESCRIPTION
**Pull request by greygoo**

### CODE REVIEW

File: plugins/services/services_inspector.rb [hunk 1]
```diff
@@ -35,10 +35,17 @@ def inspect(_filter, _options = {})
         init_system: "systemd"
       )
     elsif @system.has_command?("initctl")
-      result = ServicesScope.new(
-        inspect_ubuntu_services,
-        init_system: "upstart"
-      )
+      if @system.has_command?("chkconfig")
```
Comment (aduffeck) [inline plugins/services/services_inspector.rb]: The redhat case is covered by the `else` branch already, so wouldn't it be sufficient to replace line 37 with `elsif @system.has_command?("initctl") && !@system.has_command?("chkconfig")`?

Comment (greygoo) [inline plugins/services/services_inspector.rb]: No, as then for RHEL6 still sysvinit would be the recorded init system, which is wrong.

Comment (aduffeck) [inline plugins/services/services_inspector.rb]: Ah, right.

**Comment by aduffeck** [comment]
lgtm once the unit test passes (which it should when you rebase against the branch).

**Pull request closed without merging**